src/main/java/io/vertx/core/shareddata/impl/Checker.java

```diff
@@ -32,7 +32,6 @@ static void checkType(Object obj) {
         obj instanceof Byte ||
         obj instanceof Character ||
         obj instanceof byte[] ||
-        obj instanceof Buffer ||
         obj instanceof JsonObject ||
```

Comment:
I think that if JsonObject and JsonArray implement it, we need to remove them from this list as well